### ISSUE TITLE
Review: Simplify the spec text

### ISSUE THREAD

**Issue by ryzokuken** [tc39/intl-displaynames-v2#23]
Not a Stage 3 concern and purely editorial, but one thing I noticed was that in https://github.com/tc39/intl-displaynames-v2/blob/main/displaynames.html#L160, you use `[[<type>]]` while the statement is inside the condition `type = "language"` so you should just use `[[language]]` to simplify the spec text.

**Comment by ryzokuken** [comment]
Actually, this goes beyond the editorial problem I thought it was... the entire set of instructions at https://github.com/tc39/intl-displaynames-v2/blob/main/displaynames.html#L153-L160 go against the description of the internal slot at https://github.com/tc39/intl-displaynames-v2/blob/main/displaynames.html#L223 IIUC.

According to the latter, I get the impression that `%DisplayNames%.[[LocaleData]].[[dialects]]` is a Record that contains a single field `"language"` (why) which is another Record with the fields `"dialectName"`and `"standardName"` which are both Records with the style fields (all this is a bit poorly specified, let's fix that part up).

However, according to the spec, you are trying to access data from `%DisplayNames%.[[LocaleData]].[[dialects]].[[<dialectHandling>]].[[language]].[[language]].[[<style>]]` (while instead you should have accessed `%DisplayNames%.[[LocaleData]].[[dialects]].[[language]].[[<dialectHandling>]].[[<style>]]`.

https://github.com/tc39/intl-displaynames-v2/blob/main/displaynames.html#L223
https://github.com/tc39/intl-displaynames-v2/blob/main/displaynames.html#L153-L160

**Comment by ryzokuken** [comment]
From poorly specified, I meant that the list is out of order, which makes it really hard to follow along and there are a few grammatical mistakes here and there which could also be editorially fixed...

**Comment by FrankYFTang** [comment]
Thanks for the feedback. I will looks into this tonight.
